common/git-hooks/pre-commit

```diff
@@ -6,4 +6,4 @@
 # Invoke the "rush prettier" custom command to reformat files whenever they
 # are committed. The command is defined in common/config/rush/command-line.json
 # and uses the "rush-prettier" autoinstaller.
-node common/scripts/install-run-rush.js prettier || exit $?
\ No newline at end of file
+# node common/scripts/install-run-rush.js prettier || exit $?
```

Comment:
do you intent to disable this?

Reply:
yeah this is giving some wired auto fix, which makes the style worse lol, so not really need it